README.md

```diff
@@ -0,0 +1,116 @@
+# aptly-rest-tools
+
+**aptly-rest-tools** is a collection of Rust-based tools and libraries
+designed to interact with and extend the functionality of the [Aptly REST
+API](https://www.aptly.info/doc/api/).
+
+### `aptly-rest`
+A foundational Rust library that provides a typed interface to the Aptly
+REST API, along with utility functions for handling Debian packages and
+repository structures.
+
+### `aptly-rest-mock`
+A mock server that simulates the Aptly REST API. Enables testing tools
+and libraries without requiring a live Aptly instance.
+
+### `aptlyctl`
+A command-line tool for interacting with Aptly via its REST API. Wraps
+common operations such as snapshot creation, repository publishing,
+and cleanup into an easy-to-use CLI.
+
+### `apt2aptly`
+Mirrors an existing apt repository into an Aptly-managed
+repository. Useful for importing upstream Debian/Ubuntu repositories or
+third-party apt sources.
+
+### `obs2aptly`
+Synchronizes packages from an [Open Build Service
+(OBS)](https://openbuildservice.org/) instance into Aptly. Designed for
+teams using OBS for package builds and Aptly for distribution.
+
+### `sync2aptly`
+A Rust library providing synchronization logic between upstream sources
+and Aptly repositories. Used by both `apt2aptly` and `obs2aptly`.
+
+### `aptly-latest-snapshots`
+A lightweight HTTP API that returns the latest published snapshot for a
+given distribution. Intended for external systems to dynamically query
+the most recent publish state.
+
+
+## Installation
+
+```sh
+cargo install --locked --git https://github.com/collabora/aptly-rest-tools aptlyctl
+~/.cargo/bin/aptlyctl --help
+```
+
+## Usage
+
+Set the token as environment variable to interact with the aptly instance.
+Alternatively, it can be passed directly to `aptlyctl` with the argument `--api-token`.
+
+``` sh
+export APTLY_API_TOKEN=XXXXXXXXXXXXXXXX
+```
+
+### List repositories
+
+``` sh
+aptlyctl --api-token "$APTLY_API_TOKEN" \
```

Comment:
Please leave out `--api token "$APTLY_API_TOKEN"`, it’s picked up automatically.

Reply:
Is it better now?